sklego/meta/decay_estimator.py

```diff
@@ -31,6 +31,11 @@ def _is_classifier(self):
             ["ClassifierMixin" in p.__name__ for p in type(self.model).__bases__]
         )
 
+    @property
+    def _estimator_type(self):
```

Comment:
It's a shame that this is assumed to be a private variable in SKlearn, but yeah ... LGTM.